week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -4,11 +4,14 @@
   Create a function which:
   - Accepts an array which contains all the students' names and their attendance counts
     (see tests to confirm how this data will be structured)
-  - Returns an array containing onl the names of the who have attended AT LEAST 8 classes
+  - Returns an array containing only the names of the students who have attended AT LEAST 8 classes
 */
 
-function eligibleStudents() {
-
+function eligibleStudents(arr) {
+  let eligibleNames = arr
+    .filter(element => element[1] >= 8)
+    .map(element => element[0]);
+  return eligibleNames;
```

Comment:
This can be simplified to just return the array. For example:

```
return arr
    .filter(element => element[1] >= 8)
    .map(element => element[0]);
```

Reply:
Done.